redis-voting-app/README.md

```diff
@@ -0,0 +1,51 @@
+## Redis Voting App
+
+To get the voting app up and running use 
+
+```
+kurtosis run github.com/kurtosis-tech/awesome-kurtosis/redis-voting-app
+```
+
+If you have this repository cloned locally and are playing with the package, use
+
+```
+cd redis-voting-app
+kurtosis run .
+```
+
+Once the package is up and running use the localhost binding of port 80 for the `voting-app`; like
```

Comment:
WDYT about replacing `...of port 80` with `...of HTTP port` o `...of the API port`?